### ISSUE TITLE
Make *tag* menu option more visible

### ISSUE THREAD

**Issue by SamuXzX** [nextcloud/server#29641]
## Is your feature request related to a problem? Please describe.
At the moment, the "labels" field in the file's informations sidebar is hidden by default. To use it I have to click the three dots in the top-right corner of the sidebar, then click the filed to focus it and finally write.

![image](https://user-images.githubusercontent.com/30182251/141158149-394545a9-fad6-4c24-a043-6f1bcbcb6ce4.png)

## Describe the solution you'd like
Unless there's some reason you already discussed, I'd find it more comfortable if the field was already visibile at the opening of the sidebar. I know it's a difference of just one step, but to me this additional step seems useless.

![image](https://user-images.githubusercontent.com/30182251/141158260-99c0fc6e-d1f2-440d-9682-93317083ee99.png)

**Comment by nimishavijay** [comment]
You're right, it is hidden, but this feature isn't used much, and showing it by default may be useless as usually the tags are empty.

Rather than showing the field all the time, a better solution may be to show the field only if there are tags.

**Comment by SamuXzX** [comment]
I rarely use tags and I agree it may be not used much.
But I'll definitely use it more if it were more accessible (and if the tags management was better, but this pertains other threads).
The sidebar itself it's not used much and that the labels field isn't so big. Additionally, this function doesn't depend from anything else: is the waste of unused space so strong that we need to hide it?
If the answer is yes, I'll agree it should stay hidden.

**Comment by alimmroth** [comment]
I think tags **can** be very useful, if they are nicely supported and at hand. I think they aren't being used much (including by me) because at the moment they aren't easily accessible and lack tools needed like filtering a folder for tags.

I opened another [feature request](https://github.com/nextcloud/server/issues/59003) to allow filtering folders by tags

**Comment by schaarsc** [comment]
in Nextcloud Hub 26 Winter there is no option to show the tags of a file. (please correct me if I'm wrong)

the FilesApp only shows  the first tag, see https://github.com/nextcloud/server/issues/54946

the tags action in the three-dot menu in the sidebar does now open the tag manager, but the tag manager has no option to only show the selected tags. it would also be very cumbersome to use this as a solution.

the sidebar used to show the tags similar to what has been requested in this issue, I don't know when and why this is no longer the case.
